developer-docs/how-to-guide/How_to_upload_existing_content_using_sunbird.md

```diff
@@ -9,286 +9,241 @@ allowSearch: true
-
-#### Request Body (Publish Content API)
-
-    {
-
-     "request": {
+For the YouTube content or file url, pass it as a `fileUrl` in [Upload API]() or set the artifactUrl of the content using [Update Content API](http://www.sunbird.org/apis/content/#operation/Update%20Content)
 
-     "content": {
 
-       "lastPublishedBy": "sunbird",
+### Publishing Content using API #### 
 
-       "publishChecklist": [
-           "No Hate speech, Abuse, Violence,Profanity"
-           "No Vulgarity",
-           "No Discrimination or Defamation",
-           "Is suitable for children",
-           "Appropriate Title, Description",
-           "Correct Board, Grade, Subject, Medium",
-           "Appropriate tags such as Resource Type, Concepts",
-           "Relevant Keywords",
-           "Content plays correctly",
-           "Can see the content clearly on Desktop and App",
-           "Audio (if any) is clear and easy to understand",
-           "No Spelling mistakes in the text",
-           "Language is simple to understand"
-       ]
+After uploading the content, publish the content using [Publish Content API](http://www.sunbird.org/apis/content/#operation/Publish%20Content). 
 
-      }
-
-     }
+#### Request Body
 
+    ```
+    {
+        "request": {
+            "content": {
+                "lastPublishedBy": "sunbird",
+                "publishChecklist": [
+                    "No Hate speech, Abuse, Violence, Profanity"
+                    "No Vulgarity",
+                    "No Discrimination or Defamation",
+                    "Is suitable for children",
+                    "Appropriate Title, Description",
+                    "Correct Board, Grade, Subject, Medium",
+                    "Appropriate tags such as Resource Type, Concepts",
+                    "Relevant Keywords",
+                    "Content plays correctly",
+                    "Can see the content clearly on Desktop and App",
+                    "Audio (if any) is clear and easy to understand",
+                    "No Spelling mistakes in the text",
+                    "Language is simple to understand"
+                ]
+            }
+        }
     }
+    ```
 
-#### Response Body (Publish Content API)
+#### Response Body
 
+    ```
     {
-
-     "result": {
-
-     "node_id": "string",
-
-     "versionKey": "string"
-
-    },
-
-    "id": ""xyz.learning.content.publish",
-
-    "ver": "3.0",
-
-    "ts": "2018-09-14T10:44:43ZZ",
-
-    "params": {
-
-     "resmsgid": "d3415c1a-94c6-4dc7-9c28-3e0dac767ee3",
-
-     "msgid": "null",
-
-     "err": "null",
-
-     "status": "string",
-
-     "errmsg": "null"
-
-     },
-
-     "responseCode": "OK",<br> "result":{
-       "publishStatus": "Publish Operation for Content Id 'do_2125903396473978881177' Started Successfully!",<br>
-       "node_id": "do_2125903396473978881177"<br>
-     }<br>
-
+        "result": {
```

Comment:
I have reformatted the response, however, the response has multiple result fields. Please check and correct.

Reply:
The response body as per API reference documentation is:
{
  "result": {
    "node_id": "string",
    "versionKey": "string"
  },
  "id": "string",
  "ver": "string",
  "ts": "string",
  "params": {
    "resmsgid": "string",
    "msgid": "string",
    "err": "string",
    "status": "string",
    "errmsg": "string"
  },
  "responseCode": {}
}
In the sample provided, the result field is showing at both the top and bottom. Request you to clarify if the field from the top has to be deleted or from the bottom. Unable to understand the comment.

Reply:
A sample response body should be put here with sample values in-place of data types.

Reply:
The example should be formatted to have the result at the end of the response body.